VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -111,6 +121,11 @@ public void onPageLoaded(List<T> newItems, boolean hasMore) {
                     hasMoreItems = hasMore;
                     loading = false;
                 }
+
+                private boolean isResetCalled() {
+                    // Disregard result, since reset() was called
+                    return instanceLoadId != activeLoadId.get();
```

Comment:
Off-topic for this pull request, but using an `AtomicInteger` for abandoning loaders seems a bit excessive here, since I assume it's meant to be controlled only from the main thread. There could simply be a reference to the current callback object, which callbacks can perform an equality check with themselves against. Furthermore, there should ideally be a cancellation method in the `PageLoader` interface, although as previously discussed here, task cancellation currently does not guarantee callback abandonment.